parts/k8s/addons/secrets-store-csi-driver.yaml

```diff
@@ -282,6 +282,20 @@ status:
   conditions: []
   storedVersions: []
 ---
+{{- /* A priority class for the daemonset such that they are not */}}
+{{- /* frozen out of a node due to the node filling up with "normal" */}}
+{{- /* pods before the daemonset controller can get the daemonset */}}
+{{- /* pods to be scheduled. */}}
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: csi-secrets-store
+  labels:
+    addonmanager.kubernetes.io/mode: EnsureExists
```

Comment:
Added this label so that kube-addon-manager would interpret this resource for loading

Reply:
Thanks - sorry I missed that.